tests/template_tests/test_loaders.py

```diff
@@ -145,22 +191,20 @@ def test_missing_template_is_cached(self):
     'DIRS': [TEMPLATES_DIR],
 }])
 class RenderToStringTest(SimpleTestCase):
```

Comment:
Many of these tests look redundant with the new (and more complete) tests I added in the `template_loader` test app. What about removing them?

One thing that's missing from the new tests is passing a context in argument.